GTG/gtk/browser/main_window.py

```diff
@@ -759,6 +760,11 @@ def focus_quickentry(self, action, param):
 
         self.quickadd_entry.grab_focus()
 
+    def focus_sidebar(self, action, param):
+        """Callback to focus the sidebar widget."""
+        self.sidebar.props.visible = True
```

Comment:
I don't know / haven't reviewed the rest of the code, but I can guess that this might be problematic; isn't there a method somewhere else in the code that handles this for you? Because toggling the sidebar (if it is not already visible) might mean you need to ensure that other parts of the UI are aware as well, such as the main menubutton (aka hamburger menu).

Reply:
Yes, setting the property notifies other parts of the code:

https://github.com/getting-things-gnome/gtg/blob/e4b148b6b31bd563737bb3516d0e30a1d7551369/GTG/gtk/browser/main_window.py#L117

https://github.com/getting-things-gnome/gtg/blob/e4b148b6b31bd563737bb3516d0e30a1d7551369/GTG/gtk/browser/main_window.py#L680-L687

For the main menubutton there is a binding:
https://github.com/getting-things-gnome/gtg/blob/e4b148b6b31bd563737bb3516d0e30a1d7551369/GTG/gtk/data/main_window.ui#L218

Reply:
Oh, I didn't know our code was so fancy / well-architected ;) nevermind then!